dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/search/ror/RorSolrDataFinder.java

```diff
@@ -0,0 +1,48 @@
+package edu.harvard.iq.dataverse.search.ror;
+
+import edu.harvard.iq.dataverse.search.RorSolrClient;
+import io.vavr.control.Try;
+import org.apache.solr.client.solrj.SolrClient;
+import org.apache.solr.client.solrj.SolrQuery;
+import org.apache.solr.client.solrj.response.QueryResponse;
+
+import javax.ejb.Stateless;
+import javax.inject.Inject;
+import java.util.List;
+
+/**
+ * Solr data finder dedicated for ROR collection
+ */
+@Stateless
+public class RorSolrDataFinder {
+
+    @Inject
+    @RorSolrClient
+    private SolrClient solrClient;
+
+    public List<RorDto> findRorData(String searchPhrase) {
+        StringBuilder queryBuilder = new StringBuilder();
```

Comment:
I'm not sure if we shouldn't make some kind of escaping of solr special characters. When user will input some less common queries then it could cause solr to crash. For example something like: `fieldName:fieldValue` would cause search by `fieldName` which is not present in the index - and it would make solrClient to throw exception

Reply:
Pls check whether my new strategy is correct regarding escaping special characters.